src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -8,6 +8,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn),649</NoWarn>
     <DefineConstants Condition="'$(TargetGroup)' == 'uap'">$(DefineConstants);uap</DefineConstants>
```

Comment:
Can we change 
```
<DefineConstants Condition="'$(TargetGroup)' == 'uap'">$(DefineConstants);uap</DefineConstants>
```
to 
```
<DefineConstants Condition="'$(TargetGroup)' == 'uap' OR '$(TargetGroup)' == 'uapaot'">$(DefineConstants);uap</DefineConstants>
```
so that we can simply use `#if uap` or `#if !uap` in the code?